plugins/scripts/inject-config.js

```diff
@@ -83,8 +83,42 @@ module.exports = function injectOptimoveConfig(context) {
         console.info('Optimove: Preparing Android platform...');
         prepareAndroid(context, optimoveConfig);
     }
+
+    if (hasPlatform(context, 'ios')) {
+        console.info('Optimove: Preparing iOS platform...');
+        prepareIos(context, optimoveConfig);
+    }
 };
 
+function prepareIos(context, OptimoveConfig) {
+    const iosPath = path.join(context.opts.projectRoot, 'platforms', 'ios');
+    const files = fs.readdirSync(iosPath);
+
+    const xcodeProj = files.find(name => name.indexOf('.xcodeproj') > -1);
+    const targetName = xcodeProj.replace('.xcodeproj', '');
+
+    const configDest = path.join(
+        iosPath,
+        targetName,
+        'Resources',
+        'optimove.plist'
+    );
+
+    if (!fs.existsSync(configDest)) {
+        console.error(
+            'optimove: optimove.plist resource not found, aborting setup'
+        );
+        return;
+    }
+
+    const config = renderTemplate('optimove.xml', {
+        OPTIMOVE_CREDENTIALS: optimoveConifg.optimoveCredentials,
+        OPTIMOVE_MOBILE_CREDENTIALS: optimoveConifg.optimoveMobileCredentials
+    });
```

Comment:
- I think this should be using the `optimove.plist` template file for rendering
- keys of the dictionary should match the {{tags}} inside the template (e.g. template maybe can be updated to have `{{OPTIMOVE_MOBILE_CREDENTIALS}}` instead of `{{optimoveMobileCredentials}}`)

Reply:
do you mean something like that:

    const config = renderTemplate('optimove.xml', {
        OPTIMOVE_CREDENTIALS: optimoveConifg.OPTIMOVE_CREDENTIALS,
        OPTIMOVE_MOBILE_CREDENTIALS: optimoveConifg.OPTIMOVE_MOBILE_CREDENTIALS
    });

Reply:
I think probably more like

```
const config = renderTemplate('optimove.xml', {
    OPTIMOVE_CREDENTIALS: optimoveConifg.optimoveCredentials,
    OPTIMOVE_MOBILE_CREDENTIALS: optimoveConifg.optimoveMobileCredentials
});
```

But the plist will need updated so that the key of the dictionary here (`OPTIMOVE_CREDENTIALS`) matches the 'placeholder': `{{OPTIMOVE_CREDENTIALS}}`

Reply:
done